driver-core/src/test/functional/com/mongodb/OperationFunctionalSpecification.groovy

```diff
@@ -244,6 +245,7 @@ class OperationFunctionalSpecification extends Specification {
         def connection = Mock(Connection) {
             _ * getDescription() >> Stub(ConnectionDescription) {
                 getServerVersion() >> new ServerVersion(serverVersion)
+                getMaxWireVersion() >> getMaxWireVersion(serverVersion)
```

Comment:
Typical fix to unit tests that now rely on getMaxWireVersion.  I suppose we don't need getServerVersion() here anymore...

Reply:
Are you planning to deprecate it?

Reply:
Not sure, but if so it should be done in a separate ticket.  But could remove the mock implementations in the tests as they are no longer called.  Should I bother?

Reply:
Well they should be removed at some point - otherwise its just noise. No time like now.